aws-curl

```diff
@@ -499,6 +507,22 @@ fi
 if [ "$EC2_CREDS" = 1 ]; then
     ec2_import_creds
 fi
+get_cred_value() {
+    echo "$1" | grep "$2 =" | cut -d ' ' -f 3-
+}
+
+if [ -n "$AWS_PROFILE" ] && [ -z "$AWS_ACCESS_KEY_ID" ]; then
```

Comment:
Suggest to move body of this condition into function `profile_import_creds` that will have profile name as parameter